src/ostorlab/agent/message/proto/v3/asset/link/link.proto

```diff
@@ -51,7 +56,8 @@ message Message {
   oneof credential_oneof {
     FormCredential form_credential = 7;
     ScriptCredential script_credential = 8;
+    BasicCredential basic_credential = 9;
   }
-  optional Message parent = 9;
-  repeated LocalStorage localstorage_items = 10;
+  optional Message parent = 10;
```

Comment:
Nooooooooooooo !!!
Never change the proto number as this will result in message inconsistency between the agent with old proto and new proto. Use 11 for basic_credential